frontend/src/lib/components/FlowEditor.svelte

```diff
@@ -1,34 +1,18 @@
 <script lang="ts">
-	import type { Schema } from '$lib/common'
-	import { FlowModuleValue, type Flow } from '$lib/gen'
-	import { loadSchema } from '$lib/scripts'
-	import { emptySchema } from '$lib/utils'
 	import { faPlus } from '@fortawesome/free-solid-svg-icons'
 	import Icon from 'svelte-awesome'
+	import { get } from 'svelte/store'
 	import FlowPreview from './FlowPreview.svelte'
+	import CopyFirstStepSchema from './flows/CopyFirstStepSchema.svelte'
+	import { addModule, flowStore } from './flows/flowStore'
 	import ModuleStep from './ModuleStep.svelte'
 	import SchemaEditor from './SchemaEditor.svelte'
-	import type SchemaForm from './SchemaForm.svelte'
-
-	export let flow: Flow
 
 	let args: Record<string, any> = {}
-	let schemas: Schema[] = []
-	let schemaForms: (SchemaForm | undefined)[] = []
-
-	function addModule() {
-		schemaForms.push(undefined)
 
-		let newModule = {
-			value: { type: FlowModuleValue.type.SCRIPT, path: '' },
-			input_transform: {}
-		}
-		flow.value.modules = flow.value.modules.concat(newModule)
-		schemas.push(emptySchema())
-	}
+	const flow = get(flowStore)
```

Comment:
This is not gonna be reactive, you're better of using $flowStore everywhere you're using flow unless I'm missing something.

Reply:
Fixed.